ambry-clustermap/src/test/java/com.github.ambry.clustermap/MockDataNodeId.java

```diff
@@ -56,6 +51,32 @@ private void populatePorts(ArrayList<Port> ports) {
     }
   }
 
+  /**
+   * Set the datacenters to which ssl is enabled. If the datacenter on which this datanode resides is
+   * part of the datacenters to which ssl is enabled, then this datanode will always return an SSL port in
```

Comment:
@vgkholla : Can you elaborate a bit on your comment ? Do you mean to suggest having different clustermaps for different DCs. For instance, for DC1, the ssl list is (DC2, DC3), for DC2, the ssl list is (DC1,DC3) and for DC3 ssl list is (DC1, DC2). Are you asking why dont we do this in our tests and we just have one clustermap for all which has (DC1, D2, DC3) ?